src/main/java/relay/entities/Session.java

```diff
@@ -0,0 +1,21 @@
+package relay.entities;
+
+import com.google.type.DateTime;
+
+import java.util.ArrayList;
+
+public class Session {
+    ArrayList<AttendanceRecord> attendance;
+    String classID;
+    AttendanceCode code;
+    Instructor instructor;
+    DateTime startedAt;
```

Comment:
Don't think we should use a Google datetime. Just use the LocalDateTime type from the Java time library.  See [https://www.w3schools.com/java/java_date.asp](https://www.w3schools.com/java/java_date.asp)